source/recipecard/cardCarousel.js

```diff
@@ -1,51 +1,137 @@
+          .forward:before, .forward:after {  
+            position: absolute;
+            content: "";
+            width: 100%;
+            height: 100%;
+            top: -100%;
+            left: 0;
+            background: lightgrey;
+            z-index: -1;
+            transition: all .35s;
+          }
+          
+          .back:before, .forward:before {
+            opacity: .5;
+          }
+          
+          .back:after, .forward:after {
+            transition-delay: .25s;
+          }
+          
+          .back:hover:before, .back:hover:after,
+          .forward:hover:before, .forward:hover:after {
+            top: 0;
+          }
+          `;
+
+        this.shadowRoot.append(style);
     }
 
 
-    /*
-    * This function will be called in an event listener when the right button is clicked.
-    * The next 3 recipe cards will be shown.
-    */
-    nextCards() {
-        const cards = this.shadowRoot.querySelectorAll('recipe-card');
-        for (let i = cards.length - 4; i >= 0; i--) {
-            if (cards[i].getAttribute('style') != 'display: none') {
-                cards[i].setAttribute('style', 'display: none');
-                cards[i + 3].style.display = '';
-            }
+    // paramter is in an array of recipe cards
+    createCardCarousel(recipeCards) {
+        const forwardButton = document.createElement('a');
+        const backwardButton = document.createElement('a');
+        forwardButton.classList.add('forward')
+        backwardButton.classList.add('back')
+        forwardButton.innerHTML = '&#10095;';
+        backwardButton.innerHTML = '&#10094;';
+
+        // Add the back button to the card carousel
+        this.shadowRoot.append(backwardButton);
+        //this.shadowRoot.querySelector('.back').setAttribute('style', 'display: none');
+
+        // Set cards in the carousel
+        for (let i = 0; i < recipeCards.length; i++) {
+            this.shadowRoot.append(recipeCards[i]);
         }
-    }
 
-    /*
-    * This function will be called in an event listener when the left button is clicked.
-    * The previous 3 recipe cards will be shown.
-    */
-    prevCards() {
-        const cards = this.shadowRoot.querySelectorAll('recipe-card');
-        for (let i = 3; i < cards.length; i++) {
-            if (cards[i].getAttribute('style') != 'display: none') {
-                cards[i].setAttribute('style', 'display: none');
-                cards[i - 3].style.display = '';
+        var currentIndex = 0;
+        // gets a number of recipes that would fit on the screen
+        var numberOfrecipes = parseInt(window.innerWidth / 400);
+
+        this.displayCards(currentIndex, numberOfrecipes, recipeCards);
+
+        // Add the forward button to the card carousel
+        this.shadowRoot.append(forwardButton);
+
+        //Binds the back and forward buttons to their respective carousel's functions
+        this.shadowRoot.querySelector('.back').addEventListener('click', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8549129)
```suggestion
        this.shadowRoot.querySelector(".back").addEventListener('click', () => {
```